cli/include/ParseCLIArgs.h

```diff
@@ -15,92 +15,27 @@
 * limitations under the License.
 ********************************************************************************/
 
-#ifndef OPENSPACENET_OPENSPACENETARGS_H
-#define OPENSPACENET_OPENSPACENETARGS_H
+#ifndef OPENSPACENET_PARSECLIARGS_H
+#define OPENSPACENET_PARSECLIARGS_H
 
+#include <OpenSpaceNetArgs.h>
+#include <OpenSpaceNet.h>
 #include <boost/program_options.hpp>
-#include <utility/Logging.h>
-#include <vector/Feature.h>
-
-#define OSN_LOG(sev) DG_LOG(OpenSpaceNet, sev)
-#define MAPSAPI_MAPID  "digitalglobe.nal0g75k"
-
 
 namespace dg { namespace osn {
 
-enum class Source
-{
-    UNKNOWN,
-    LOCAL,
-    DGCS,
-    EVWHS,
-    MAPS_API,
-    TILE_JSON
-};
-
-enum class Action
-{
-    UNKNOWN,
-    HELP,
-    DETECT,
-    LANDCOVER
-};
-
-class OpenSpaceNetArgs
+class ParseCLIArgs
```

Comment:
You may want to reconsider this name.  It's an argument parser.  Class names really should be nouns, not verbs.